plugins/BEdita/Core/tests/TestCase/Utility/JsonSchemaTest.php

```diff
@@ -227,8 +229,12 @@ public function testGenerate($expected, $name)
         } else {
             static::assertNotEmpty($result);
 
-            $keys = ['definitions', '$id', '$schema', 'type', 'properties', 'required', 'revision'];
-            static::assertEquals($keys, array_keys($result), '', 0, 10, true);
+            $keys = ['definitions', '$id', '$schema', 'type', 'properties', 'required', 'associations', 'relations', 'revision'];
+            if (!in_array($name, ['roles', 'streams'])) {
+                $keys = array_merge($keys, ['associations', 'relations']);
```

Comment:
Why do we merge `['associations', 'relations']` that are already in `$keys`?

Reply:
No reason :wink: - I'll remove the condition